kasa/discover.py

```diff
@@ -273,9 +273,16 @@ async def discover_single(
         """
         loop = asyncio.get_event_loop()
         event = asyncio.Event()
+        adrrinfo = await loop.getaddrinfo(
+            host,
+            0,
+            type=socket.SocketKind.SOCK_DGRAM,
+            family=socket.AddressFamily.AF_INET,
+        )
+        ip = adrrinfo[0][4][0]
```

Comment:
```suggestion
        adrrinfo = await loop.getaddrinfo(
            host,
            0,
            type=socket.SocketKind.SOCK_DGRAM,
            family=socket.AddressFamily.AF_INET,
        )
        ip = adrrinfo[0][4][0]
```

`addrinfo`

If its already an ip, we should try to avoid the lookup

Reply:
Done